### DISCUSSION TITLE
Has the CLI still a future?

### DISCUSSION THREAD

**Discussion by jasperslits** [uilibs/uiprotect#13] [General]
I am wondering if the CLI should be retained or just moved to its own repo as the main objective - to me - should be to provide a library to integrate with Home Assistant. 
Maintaining the CLI increases the burden and if one has a need for it, using the current pyunifiprotect package does the job. 

It may help with testing new functionalities outside but my experience when adding support for Unifi OS 4 token changes, that the CLI behaved different compared to HA for authentication. 

Thoughts?

**Comment by bdraco** [comment]
I'm not sure what the best route is here. The CLI is still useful for testing. Its a bit of a double edged sword as it might be more work to test if we remove it here, but the cost to maintain it is a bit of work as well.

**Comment by ep1cman** [comment]
> using the current pyunifiprotect package does the job.

It got deleted from github and PyPI today sadly

**Comment by bdraco** [comment]
Looks like we need to keep it given this is the only active branch
